Frameworks/UIKit/UIPanGestureRecognizer.mm

```diff
@@ -541,4 +528,8 @@ + (BOOL)_fireGestures:(id)gestures shouldCancelTouches:(BOOL&)shouldCancelTouche
     return didRecognize;
 }
 
+- (const std::vector<TouchInfo>&)_getTouches {
```

Comment:
you mean StrongId<WUXIPointerRoutedEventArgs> _routedEventArgs;? StrongId here is to make sure UITouch has the same life time for routedEventArgs, as we see in multiple-finger gesture cases, routedEventArgs can be become invalid though UITouch is still valid. Still a mystery to figure out why. But copy won't solve this problem either plus add additional cost.  

---
In reply to: [86604604](https://github.com/Microsoft/WinObjC/pull/1305#discussion_r86604604) [](ancestors = 86604604)